### PR TITLE
[Feat]: Created web responsive header

### LINKED ISSUE #21: [Feat]: Add a Responsive Header Component
## Description
[Initial Designs](https://www.figma.com/design/1nsYgWIN5bC7NnnVApTcK4/Salinas---bizzNest-Scheduler?node-id=0-1&node-type=canvas&t=TLQ1O5AYgPPOLJtm-0) design

### Web
image

### Mobile
image

> \[!NOTE]\
>  Every single UI PR should consist of a Screenshot and/or video (You can use cmd+shift+4 for screenshots and cmd+shift+5 for video recordings) and should be mobile responsive (show the mobile view)

### Tasks

- [x] Create a header with proper semantics tags
- [x] Have the correct background color
- [x] Have the correct font
- [x] Have the Digital Nest Logo (note the logo for mobile is different)
- [x] Have the content properly aligned (keep title center, logo left ... etc)

### Additional Resources + Notes
Good to look into CSS + Semantic Tags + HTML

### PR DESCRIPTION
**Pull request by jacky-p**
## Changes
**1. Added header to HTML compatible for Desktop view**
Screen Shot 2024-09-24 at 2 50 32 PM

**3. Added header compatibility with mobile, changes to mobile view once the screen is a size of 550px or less.**
Screen Shot 2024-09-24 at 2 42 21 PM

## Purpose
Provides UI for the header for the scheduler.

## Approach
Added header features to `index.html` and styled the header with CSS in `style.css` based on our [figma design](https://www.figma.com/design/1nsYgWIN5bC7NnnVApTcK4/Salinas---bizzNest-Scheduler?node-id=5-2986&t=skfSGNXewze1jfpO-1) .

## Testing Steps
By looking at the views provided via desktop and mobile to see if it displays as it should.

## Learning
To follow the fonts used in out figma design we had to import fonts to be used on the website
- [Resource for linking/loading/using custom fonts](https://www.digitalocean.com/community/tutorials/how-to-load-and-use-custom-fonts-with-css)
- ["Outfit" Google font used for web name](https://fonts.google.com/selection/embed)

Responsive Header Design Resources
- [On displaying varying images on varying screen sizes] (https://stackoverflow.com/questions/28966158/changing-image-depending-on-mobile-or-desktop-html-css)
- [Tips on responsive web design] (https://www.w3schools.com/html/html_responsive.asp)
- [Media Queries] (https://www.w3schools.com/html/html_responsive.asp)

Closes #21

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,4 +1,72 @@
...
+  justify-content: space-between;
+  background-color: #BDE3D0;
+  width: 100%;
+
+}
+
+.logo {
+  padding: 20px;
+  align-self: center;
+
```
Comment (elimanzo) [inline src/style.css:25]:
```suggestion
```

here as well :)

File: src/style.css [hunk 2]
```diff
@@ -1,4 +1,72 @@
...
+  font-style: normal;
+}
+
+.web-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #BDE3D0;
+  width: 100%;
+
```
Comment (elimanzo) [inline src/style.css:19]: Couple of unwanted white spaces
```suggestion
```

**Review by elimanzo — APPROVED** [review]
Good Job! just small nit and we're ready to go.

File: src/style.css (hunk 2 above)
Comment (jacky-p) [reply src/style.css]: Okay! Noted and resolved :)

**Review by lilianaconstantino — APPROVED** [review]
Awesome!!

**Pull request merged**